utils/parts.stanza

```diff
@@ -141,8 +141,9 @@ public-when(TESTING) defstruct NoConnectCode :
 with :
   printer => true
 
-public-when(TESTING) defstruct VendorPartNumbersCode :
-  lcsc: String|UNKNOWN
+public-when(TESTING) defstruct VendorPartNumberCode :
+  exists: True
```

Comment:
If the `exists` field is always True maybe it is not needed?

Reply:
Reading your PR description I think I understand better.
Perhaps use a `True|False` type even if it is only populated with True for now?

Reply:
:+1: